docs/articles/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+title: Running Tests Using Travis CI and Sauce Labs
+permalink: /documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.html
+---
+# Running Tests Using Travis CI and Sauce Labs
+
+You can automatically run tests as a part of your build process using TestCafe and [Travis CI](https://travis-ci.org/).
+Furthermore, Travis CI integrates with [Sauce Labs](https://saucelabs.com/) that provides access to multiple cloud browsers.
```

Comment:
It's not true. Travis CI doesn't integrates with SauceLabs. You just set env variables via Travis UI. But TestCafe allows to run tests on SauceLabs out-the-box